.circleci/config.yml

```diff
@@ -28,9 +46,23 @@ jobs:
     steps:
       - checkout
       - security/scan_dependencies:
-          pkg_manager: npm
           pkg_json_dir: <<parameters.pkg_json_dir>>
-
+  test-components:
+    machine:
+      image: ubuntu-2204:current
+    steps:
+      - checkout
+      - aws-cli/setup:
+          role_arn: ${AWS_ROLE_ARN}
+      - setup-pulumi
+      - node/install
+      - core/install_dependencies
+      - core/install_dependencies:
+          pkg_json_dir: './tests/web-server/infrastructure'
```

Comment:
@mbareta @droguljic this will need to be updates when we have workspaces in place. Will it enable us to perform install once for all component tests?